.distignore

```diff
@@ -1,2 +1,49 @@
-node_modules
-.github
\ No newline at end of file
+/.git
+.gitignore
+.gitattributes
+
+/.idea
+/.github
+/.storybook
+/.vscode
+/.wordpress-org
+
+/.husky
+/docs
+/node_modules
+/src
+/storybook
+/tests
+/types
+/vendor/bin
+/vendor/composer/installers
+/vendor/**/*.phar
+/wp-feature-notifications
+
+.distignore
+.editorconfig
+.eslintignore
+.eslintrc.js
+.gitattributes
+.gitignore
+.lintstagedrc
+.nvmrc
+.prettierignore
+.prettierrc.js
+.stylelintrc
+.wp-env.json
+
+babel.config.js
+composer.json
+composer.lock
+jest.config.js
+package-lock.json
+package.json
+phpcs.xml.dist
+phpunit.xml.dist
+README.md
```

Comment:
I think it's worth having a readme in the distribution zip, personally, although I agree with the rest. What do you think?

Reply:
What is excluded is not the readme.txt (the one used by WordPress) but the one that is used by GitHub, ad IMHO it is ok like this because otherwise it would be a "duplicate"

Reply:
🤦 oh that's a good point. Feel free to disregard, I think that makes sense.

Reply:
@Sephsekla I drew a lot from the GiveWP repo, they exclude the readme markdown file too.